proto/api/v2/compliance_common.proto

```diff
@@ -0,0 +1,28 @@
+syntax = "proto3";
+
+package v2;
+
+option go_package = "v2";
+option java_package = "io.stackrox.proto.api.v2";
+
+message ComplianceRule {
+  string name = 1;
+  reserved 2;
+  string rule_type = 3;
+  string severity = 4;
+  string standard = 5;
+  string control = 6;
+  string title = 7;
+  string description = 8;
+  string rationale = 9;
```

Comment:
This data is coming from Compliance Operator, right?
This might get confusing to differentiate between Compliance Operator data & Benchmark data when looking at the source :cold_sweat: 
Not blocking this PR, just noting.